tests/components/onewire/test_entity_owserver.py

```diff
@@ -194,11 +427,22 @@ async def test_owserver_setup_valid_device(hass, device_id):
 
     assert len(entity_registry.entities) == len(expected_sensors)
 
+    if len(expected_sensors) > 0:
+        device_info = mock_device_sensor["device_info"]
+        assert len(device_registry.devices) == 1
+        registry_entry = device_registry.async_get_device({(DOMAIN, device_id)}, set())
+        assert registry_entry is not None
+        assert registry_entry.identifiers == {(DOMAIN, device_id)}
+        assert registry_entry.manufacturer == device_info["manufacturer"]
+        assert registry_entry.name == device_info["name"]
+        assert registry_entry.model == device_info["model"]
+
     for expected_sensor in expected_sensors:
         entity_id = expected_sensor["entity_id"]
         registry_entry = entity_registry.entities.get(entity_id)
         assert registry_entry is not None
         assert registry_entry.unique_id == expected_sensor["unique_id"]
         assert registry_entry.unit_of_measurement == expected_sensor["unit"]
+        assert registry_entry.device_class == expected_sensor["class"]
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6372789)